src/dev/src/app/quick-list/quick-list.demo.html

```diff
@@ -2,15 +2,18 @@ <h2>Quick List</h2>
 <div class="clr-example">
   <div class="main-container">
     <div class="clr-row">
```

Comment:
main-container and clr-row are destroying the layout in the demo. pls remove them and test if it looks the same (margins, paddings) as in the combobox demo